src/helpers/repetitions.js

```diff
@@ -49,7 +50,40 @@ const ascendingSortStartHour = (a, b) => {
   return 1;
 };
 
-const groupRepetitionsByDay = (repetitions) => {
+const getConflictsInfo = (query, repetitionsGroupedByDay) => {
+  const repetitionsByDayWithConflictInfos = cloneDeep(repetitionsGroupedByDay);
+  if (get(query, 'auxiliary')) {
+    for (const repetitionList of Object.values(repetitionsByDayWithConflictInfos)) {
+      for (let i = 0, l = repetitionList.length; i < l; i++) {
```

Comment:
On écrit plutot nos boucles for comme ca non ? 
`for (const i = 0l i < repetitionList.length; i ++)`

Reply:
J'ai l'impression qu'on utilise beaucoup la notation que j'ai utilisé lorsque l'on boucle sur les index

Reply:
Ha oui tu as raison, je n'avais pas fait gaffe que c'etait pas mal utilisé dans notre code 🤔 
Quels avantages vois-tu à cet écriture ?

Reply:
En soit les deux sont équivalentes et celle que tu proposes est plus concise donc je suis d'avis a utiliser `for (let i = 0; i < repetitionList.length; i ++)`